lib/jekyll_auth/config.rb

```diff
@@ -3,21 +3,36 @@ def self.config_file
     File.join(Dir.pwd, '_config.yml')
   end
 
+  def self.jekyllConfig
```

Comment:
Ruby conventions dictate that both the method and the instance variable should be `jekyll_config`.